tests/tekton-pipelines_test.go

```diff
@@ -33,25 +34,25 @@ var _ = Describe("Tekton Pipelines Operand", func() {
 			strategy.RevertToOriginalSspCr()
 		})
 
-		It("[test_id:TODO] should create pipelines", func() {
-			pipelineList := &pipeline.PipelineList{}
-
-			Eventually(func() bool {
-				err := apiClient.List(ctx, pipelineList,
-					client.MatchingLabels{
-						common.AppKubernetesManagedByLabel: common.AppKubernetesManagedByValue,
-						common.AppKubernetesComponentLabel: string(common.AppComponentTektonPipelines),
-					},
-				)
-				Expect(err).ToNot(HaveOccurred())
-				return len(pipelineList.Items) > 0
-			}, env.ShortTimeout(), time.Second).Should(BeTrue())
-
-			for _, pipeline := range pipelineList.Items {
-				Expect(pipeline.Labels[common.AppKubernetesManagedByLabel]).To(Equal(common.AppKubernetesManagedByValue), "managed by label should equal")
-				Expect(pipeline.Labels[common.AppKubernetesComponentLabel]).To(Equal(string(common.AppComponentTektonPipelines)), "component label should equal")
-			}
-		})
+		// It("[test_id:TODO] should create pipelines", func() {
```

Comment:
In the future, you can disable tests by setting them to "pending", using `PIt()` instead of `It()`. Then we don't need to comment out code.